src/DssSpell.t.sol

```diff
@@ -36,21 +36,132 @@ contract DssSpellTest is DssSpellTestBase {
         checkCollateralValues(afterSpell);
     }
 
+    function giveTokensGUSD(DSTokenAbstract token, uint256 amount) internal {
```

Comment:
this could be generalized by overloading the `giveTokens` function, adding the `store` parameter, and moved to base tests in case we may need in the future.

Reply:
I'm hesitant to generalize this as I don't know of any other tokens that do this. In all likelyhood this is a one-off.